egs/wsj/s5/local/nnet3/run_lstm.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+# this is a basic lstm script
+# LSTM script runs for more epochs than the TDNN script
+# and each epoch takes twice the time
+
+# At this script level we don't support not running on GPU, as it would be painfully slow.
+# If you want to run without GPU you'd have to call lstm/train.sh with --gpu false
+
+stage=0
+train_stage=-10
+affix=
+common_egs_dir=
+
+# LSTM options
+splice_indexes="-2,-1,0,1,2 0 0"
+label_delay=5
+num_lstm_layers=3
+cell_dim=1024
+hidden_dim=1024
+recurrent_projection_dim=256
+non_recurrent_projection_dim=256
+chunk_width=20
+chunk_left_context=40
+clipping_threshold=30.0
+norm_based_clipping=true
+
+num_epochs=10
```

Comment:
We normally use 8 epochs for TDNN training with WSJ data. From other recipes we can see that LSTM is converging slowly compared to the TDNN recipe. I would suggest increasing the number of epochs for LSTM and BLSTM. Try 16. Plot the training and CV accuracies from the compute_prob_*.logs and check if the accuracies have saturated.